enumfields/fields.py

```diff
@@ -128,11 +128,24 @@ def formfield(self, form_class=None, choices_form_class=None, **kwargs):
 
 
 class EnumField(EnumFieldMixin, models.CharField):
-    def __init__(self, enum, **kwargs):
-        kwargs.setdefault("max_length", 10)
+    def __init__(self, enum, auto_length=False, **kwargs):
+        max_item_length = self._get_max_item_length(enum)
+        kwargs.setdefault("max_length", (max_item_length // 10 + 1) * 10 if auto_length else 10)
+
+        assert max_item_length <= kwargs["max_length"], 'The enum have value longer that max_length of the field!'
+
         super(EnumField, self).__init__(enum, **kwargs)
         self.validators = []
 
+    @staticmethod
+    def _get_max_item_length(enum):
```

Comment:
There's no reason for this to be a class staticmethod; just make it a free function instead?

Reply:
Can I add such method to `Enum` class?